Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -481,7 +481,22 @@ private void MeleeDamage(FPSWeapon weapon)
 
                 // TODO: Use correct damage based on weapon and swing type
                 // Just using fudge values during development
```

Comment:
From what I saw in the loot generator, most of the other weapons data are stored in hard coded enums in the project. Even if the goal was to import all the information from DFVanilla, I think adding more data tables is an acceptable first step just to get the damage formulas in and tested. 

@Interkarma what are your thoughts?